modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/genitals.dm

```diff
@@ -7,41 +7,45 @@
 	///Where the genital is on the body. If clothing doesn't cover it, it shows up!
 	var/genital_location = GROIN
 
-/datum/sprite_accessory/genital/is_hidden(mob/living/carbon/human/H, obj/item/bodypart/HD)
-	var/obj/item/organ/external/genital/badonkers = H.getorganslot(associated_organ_slot)
+/datum/sprite_accessory/genital/is_hidden(mob/living/carbon/human/target_mob, obj/item/bodypart/genital_to_hide)
```

Comment:
It's really stupid, but that argument is the actual head of the target mob. Honestly we should just get rid of it entirely, but that's not really in the scope of this PR.
```suggestion
/datum/sprite_accessory/genital/is_hidden(mob/living/carbon/human/target_mob, obj/item/bodypart/head)
```

Reply:
Why in the world does the mob's head get called on this lmao
I had no clue what HD stood for, nice to know it actually DID mean something that makes a little sense